test/test.js

```diff
@@ -256,4 +256,154 @@ describe('ConfigGenerator', function () {
+
+            var actualPath = path.resolve(__dirname, 'modal/js/namespace-define-change.es.js');
+            var originalContent = fs.readFileSync(path.resolve(__dirname, 'modal/js/namespace-define-original.es.js'), 'utf-8')
+            var actual = fs.readFileSync(actualPath, 'utf-8');
+            var expected = fs.readFileSync(path.resolve(__dirname, 'expected/expected-namespace-define-change.es.js'), 'utf-8');
+            assert.strictEqual(normalizeCR(actual), normalizeCR(expected));
+
+            fs.writeFileSync(actualPath, originalContent);
+            done();
+        });
+    });
+
+    it('should rewrite "require" calls if "namespace" option is present', function (done) {
+        var configGenerator = new ConfigGenerator({
+            args: [path.resolve(__dirname, 'modal')],
+            config: '',
+            filePattern: '**/namespace-require-skip-override*.js',
+            format: ['/_/g', '-'],
+            ignorePath: false,
+            moduleConfig: path.resolve(__dirname, 'modal/bower.json'),
+            moduleRoot: path.resolve(__dirname, 'modal'),
+            skipFileOverride: true,
+            namespace: 'Liferay'
+        });
+
+        configGenerator.process().then(function(config) {
+            var actual = fs.readFileSync(path.resolve(__dirname, 'modal/js/namespace-require-skip-override.es.js'), 'utf-8');
+            var expected = fs.readFileSync(path.resolve(__dirname, 'expected/expected-namespace-require-skip-override.es.js'), 'utf-8');
+            assert.strictEqual(normalizeCR(actual), normalizeCR(expected));
+
+            done();
+        });
+    });
+
+    it('should rewrite "require" calls and save the file if "namespace" option is present and "skipFileOverride" option is "false"', function (done) {
+        var configGenerator = new ConfigGenerator({
+            args: [path.resolve(__dirname, 'modal')],
+            config: '',
+            filePattern: '**/namespace-require-override*.js',
+            format: ['/_/g', '-'],
+            ignorePath: false,
+            moduleConfig: path.resolve(__dirname, 'modal/bower.json'),
+            moduleRoot: path.resolve(__dirname, 'modal'),
+            skipFileOverride: false,
+            namespace: 'Liferay'
+        });
+
+        configGenerator.process().then(function(config) {
+            var actual = fs.readFileSync(path.resolve(__dirname, 'modal/js/namespace-require-override.es.js'), 'utf-8');
+            var expected = fs.readFileSync(path.resolve(__dirname, 'expected/expected-namespace-require-override.es.js'), 'utf-8');
+            assert.strictEqual(normalizeCR(actual), normalizeCR(expected));
+            done();
+        });
+    });
+
+    it('should rewrite "require" calls if "namespace" option is present and change the "namespace" if different', function (done) {
```

Comment:
Hi @jbalsas,

Sorry for the confusion. 

I updated code as well as the tests to only re-write "pure" `define` and `require` calls.